### DISCUSSION TITLE
Best databases for mobile apps ?

### DISCUSSION THREAD

**Discussion by LucssCarvalho** [community/community#38018] [Mobile] [answered]
Firebase, Realm, SQLite, MongoDB, Amazon DynamoDB, etc ...

**Comment by david-engelmann** [comment, marked as answer]
Choosing a database can definitely be difficult. It really comes down to what you are storing, how much of it you are storing and how much you'd like to pay to store it. MySql is usually the easiest and cheapest option if relational will work. The cloud based relational databases offer the benefits of cloud in persistence and access but come at a higher price. If you have a ton of unstructured data then MongoDB or other no-sql options could make more sense!

**Comment by nao1247** [comment]
> Firebase, Realm, SQLite, MongoDB, Amazon DynamoDB, etc ... 
![image](https://user-images.githubusercontent.com/121165080/209486978-c31f20c1-aa5d-44f1-aac5-21dc712462f9.png)
